src/Tack/Tack.cs

```diff
@@ -27,8 +28,9 @@ protected override void ConfigureServices(IServiceCollection serviceCollection,
 
         protected override Task<int> Run(Options options)
         {
-            MSBuildLocator.RegisterDefaults();
-
+            var instance = MSBuildLocator.RegisterDefaults();
+            Log.Information($"Runtime version:{Environment.Version}, MSBuildPath:{instance.MSBuildPath}");
```

Comment:
I think that logging the msbuild and runtime can be very helpful (Especially if something doesn't work).